ts/src/abstract/btse.ts

```
@@ -0,0 +1,33 @@
+    privateGetApiV32UserWallet (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserWalletSymbol (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserOpenOrders (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserOrders (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserOrderOrderId (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserTrades (params?: {}): Promise<implicitReturnType>;
+    privateGetApiV32UserTradesSymbol (params?: {}): Promise<implicitReturnType>;
+    privatePostApiV32Order (params?: {}): Promise<implicitReturnType>;
+    privateDeleteApiV32Order (params?: {}): Promise<implicitReturnType>;
+    privateDeleteApiV32OrderOrderId (params?: {}): Promise<implicitReturnType>;
+}
+abstract class Exchange extends _Exchange {}
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)